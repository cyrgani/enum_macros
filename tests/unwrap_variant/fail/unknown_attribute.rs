use enum_macros::UnwrapVariant;

#[derive(UnwrapVariant)]
#[allow(dead_code)]
enum Test {
    #[unwrap(xyz)]
    A(String),
    B(usize),
}

fn main() {}