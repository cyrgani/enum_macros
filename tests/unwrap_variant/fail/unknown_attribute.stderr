error: unrecognized unwrap attribute
 --> tests/unwrap_variant/fail/unknown_attribute.rs:6:14
  |
6 |     #[unwrap(xyz)]
  |              ^^^
